open-craft/edx-platform
# [SE-4953] Make course description editable in certs

This PR makes the default description text that appears under the course title editable/configurable for certificates.

**JIRA tickets:** [SE-4953](https://tasks.opencraft.com/browse/SE-4953)

**Testing Instructions:**
1. Check out this branch on your devstack.
2. Enable certificates on the demo course.
3. On clicking "Set up your certificate", you should see the additional field "Course Description".
4. Add in some text for course description and preview certificate.
5. Verify that the custom course description is shown on the certificate. 

Note: This is already deployed to Esme's cambridge instance, so you can test it against the Demo course there (or create a new one for testing).

**Reviewers:**
